spec/Schema.spec.js

```diff
@@ -178,6 +178,38 @@ describe('SchemaController', () => {
     });
   });
 
+  it('class-level permissions test count', (done) => {
+    var obj;
+    return config.database.loadSchema()
+      // Create a valid class
+      .then(schema => schema.validateObject('Stuff', {foo: 'bar'}))
+      .then(schema => {
+        var count = {};
+        return schema.setPermissions('Stuff', {
+          'create': {'*': true},
+          'find': {'*': true},
+          'count': count
```

Comment:
yeah but *: false is not a valid CLP, {} means lockdown, no one can read / write